python/py-soco/Portfile

```diff
@@ -35,9 +35,15 @@ if {${name} ne ${subport}} {
 
     depends_test-append \
                     port:py${python.version}-pip \
+                    port:py${python.version}-pyflakes \
+                    port:py${python.version}-pytest \
+                    port:py${python.version}-requests-mock \
                     port:py${python.version}-wheel
 
     test.run        yes
+    test.cmd        py.test-${python.branch} -m 'not integration'
```

Comment:
ideally the `-m 'not integration' should be set as `test.args` and not be part of the `test.cmd`